src/craftdef.cpp

```diff
@@ -157,32 +155,27 @@ static bool craftGetBounds(const std::vector<std::string> &items, unsigned int w
 	bool success = false;
 	unsigned int x = 0;
 	unsigned int y = 0;
-	for(std::vector<std::string>::const_iterator
+	for (std::vector<std::string>::const_iterator
 			i = items.begin();
-			i != items.end(); i++)
```

Comment:
While you're at it, I prefer that iterators be named `it` or even `iter` because `i` suggests an integer.  I don't think that's officially in the guidelines though.